clients/src/main/java/org/oracle/okafka/common/utils/TimestampConverter.java

```diff
@@ -0,0 +1,32 @@
+package org.oracle.okafka.common.utils;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+public class TimestampConverter {
```

Comment:
why do we need it this class ?

Reply:
because in our database tables the enqueue timestamp is in UTC format like dd-MMM-yy hh.mm.ss.SSSSSS a XXX but in kafka we have to return and pass the timestamp in UNIX epoch in milliseconds